<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vPartProcess</class>
 <widget class="QWidget" name="vPartProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>730</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>新細明體</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tbProcess">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>591</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTreeWidget" name="trProduct">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>591</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbProducts">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>571</width>
     <height>591</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsert">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRelist">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Relist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlot">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsert2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemove2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRelist2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Relist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTypeSelect">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TypeSelect</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtProduct">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>330</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProcessImport">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>670</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>製程代碼</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
